### PR TITLE
Implement CH4 challenge 2-Address generation

### PR DESCRIPTION
**Pull request by sabakhilji**
#626

for testing. I made some changes in the address-1 file to implement logic in the Python terminal.

[This PR has not been implemented completely]

### CODE REVIEW

File: content/lessons/chapter-4/address-1.tsx [hunk 1]
```diff
@@ -46,30 +46,30 @@ const javascript = {
...
+# Get the sha256 digest of the compressed public key.
+# Then get the ripemd160 digest of that sha256 hash
+# Return 20-byte array
 # Type your code here
 `,
   validate: async (answer) => {
-    if (!answer.startsWith('00000')) {
-      return [false, 'Hash must start with 5 zeroes.']
+   
+    if (len(answer)!==20) {
```
Comment (benalleng) [inline content/lessons/chapter-4/address-1.tsx:60]: this should be `if (answer.length !== 20) {` remember even though this is checking the result of python code we are still using JS! Also if you are failing to compile remember that you can always run `yarn build` or `yarn lint` to check code quality

**Review by benalleng — CHANGES_REQUESTED** [review]

File: content/lessons/chapter-4/address-1.tsx (hunk 1 above)
Comment (sabakhilji) [reply content/lessons/chapter-4/address-1.tsx]: Thanks @benalleng it worked!!

**Review by benalleng — CHANGES_REQUESTED** [review]
Coming along, a small length issue that might be stopping you

**Review by benalleng — APPROVED** [review]
TACK! LGTM

we will need to create a new PR to allow for code from @0tuedon 's challenge but I am happy with this ATM

**Pull request merged**